Modix.Services/Notes/NoteRetrieverService.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace Modix.Services.Notes
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Data;
+    using Data.Models.Moderation;
+    using Serilog;
+
+    public class NoteRetrieverService : INoteRetrieverService
+    {
+        private readonly ModixContext _modixContext;
+
+        public NoteRetrieverService(ModixContext modixContext)
+        {
+            _modixContext = modixContext;
+        }
+
+        public List<NoteEntity> FindNotesForUser(ulong userId)
+        {
+            try
+            {
+                var notes = _modixContext.Notes.Where(n => n.UserId == userId).ToList();
+                return notes;
```

Comment:
It's probably worth ordering by when the note was created so that the newest notes are at the top.